recipes/cpython/all/conanfile.py

```diff
@@ -647,6 +712,12 @@ def package_info(self):
         )
         self.cpp_info.components["embed"].requires = ["python"]
 
+        # Transparent integration with CMake's FindPython(3)
+        self.cpp_info.set_property("cmake_file_name", "Python3")
+        self.cpp_info.set_property("cmake_module_file_name", "Python")
```

Comment:
Too bad there's no way to create file name aliases in Conan.
This will break for `find_package(Python3 MODULE)`, but it's rarely used that way, fortunately.

Reply:
Yea, that would be useful here. I saw Freetype used a similar pattern so that's how I did it.

I'm not exactly sure what `MODULE` mode does differently anyways, or if FindPython(3) even supports it.

Reply:
`MODULE` simply excludes `*-config.cmake`/`*Config.cmake` and only allows `Find*.cmake` to be used by `find_package()`.

Reply:
Ah, yea that's probably rare enough to not worry about. I could also swap them if for whatever reason we think `find_package(Python3 MODULE)` would be more common than `find_package(Python MODULE)`.